### DISCUSSION TITLE
Issue 718 not resolved [Mar 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5436] [Discord Threads]
> 🙋‍♂️ **sebigboss**

**Comment by o1lab** [comment]
> 🙋‍♂️ **sebigboss**  
#### Just to provide all the info: database is postgresql 11, user used to create nocodb project is NOT the owner (because I need to manage permissions), but some tables need to be editable which they are not because you cannot edit a table without primary key.  
#  
> 🙋‍♂️ **sebigboss**  
#### Seems to be fine if the user has any write rights on some tables. Sorry for messing up.  
#  
> 🙋‍♂️ **sebigboss**  
#### (Got the confusing message when connecting with a read-only user.)  
#  
> 🙋‍♂️ **kyleecodes**  
#### Tagging <@&682859083799199814>    
####    
#### https://github.com/nocodb/nocodb/issues/719  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1080244368973234266)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
